### ISSUE TITLE
 I would like to ask whether this code has relevant papers

### ISSUE THREAD

**Issue by Ailiennl** [haoyye/ResourceAllocationReinforcementLearning#1]
Excuse me, I would like to ask whether this code has relevant papers

**Comment by zohaibshabbir** [comment]
Hi Ailiennl, 
did you tried to run this code ?

**Comment by Ailiennl** [comment]
Yes, but I found it difficult to understand your code, so I would like to ask if you can add a description of this code.




------------------ 原始邮件 ------------------
发件人: "zohaibshabbir"<notifications@github.com>; 
发送时间: 2019年5月22日(星期三) 下午5:15
收件人: "haoyye/ResourceAllocationReinforcementLearning"<ResourceAllocationReinforcementLearning@noreply.github.com>; 
抄送: "Xidian Student"<286914224@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [haoyye/ResourceAllocationReinforcementLearning]  I would like toask whether this code has relevant papers (#1)




Hi Ailiennl,
 did you tried to run this code ?

**Comment by zohaibshabbir** [comment]
Thanks for a response. I tried to run this code but unable to get any graph. 
did you get any graph after running this code ?

**Comment by Ailiennl** [comment]
I also can't get a picture.




------------------ 原始邮件 ------------------
发件人: "zohaibshabbir"<notifications@github.com>; 
发送时间: 2019年5月22日(星期三) 晚上8:18
收件人: "haoyye/ResourceAllocationReinforcementLearning"<ResourceAllocationReinforcementLearning@noreply.github.com>; 
抄送: "Xidian Student"<286914224@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [haoyye/ResourceAllocationReinforcementLearning]  I would like toask whether this code has relevant papers (#1)




Thanks for a response. I tried to run this code but unable to get any graph.
 did you get any graph after running this code ?

**Comment by ZXHL** [comment]
Hello, I have a question for you. What does up_lane、up_lane, left_lane, right_lane, width, height mean in the paper?
![62107096-79a4c680-b2d9-11e9-8c69-8fb69693fe9a](https://user-images.githubusercontent.com/48380516/64508808-e071c280-d310-11e9-9cea-0c1305ac2cc8.png)

**Comment by tangtangsiqi** [comment]
I also want to ask whether relevant papers can be shared.

**Comment by Engineer1999** [comment]
Hello all, this code is for the research article "Deep Reinforcement Learning based Resource
Allocation for V2V Communications" by Hao Ye, Geoffrey Ye Li, and Biing-Hwang Fred Juang. By this repository with some modification, I am been able to reproduce the results of this research paper.
Link:- [https://ieeexplore.ieee.org/document/8633948](url)

**Comment by Engineer1999** [comment]
Please go to this link for research article:- https://ieeexplore.ieee.org/document/8633948
